tests/warning_examples/invalid-numeric-range-parameter-default.yaml

```diff
@@ -0,0 +1,15 @@
+---
+
+- step:
+    name: test
+    image: tensorflow/tensorflow
+    command: python -u /tmp/code/my_mnist.py
+    parameters:
+      - default:
+        - 1337
```

Comment:
Not related to this pr, but how come default is a list when the type is integer? What am I missing, I'd expect it to be an integer.